toxcore/tox.h

```diff
@@ -1065,14 +1064,16 @@ typedef void tox_self_connection_status_cb(Tox *tox, Tox_Connection connection_s
 void tox_callback_self_connection_status(Tox *tox, tox_self_connection_status_cb *callback);
 
 /**
- * @brief Return the time in milliseconds before `tox_iterate()` should be called again
- *   for optimal performance.
+ * @brief Return the time in milliseconds before `tox_iterate()` should be
+ *   called again for optimal performance.
  */
 uint32_t tox_iteration_interval(const Tox *tox);
 
 /**
- * @brief The main loop that needs to be run in intervals of `tox_iteration_interval()`
- *   milliseconds.
+ * @brief The main loop that needs to be run in intervals of
+ * `tox_iteration_interval()` milliseconds.
```

Comment:
Is there a reason not to indent the `@brief`?

Reply:
Missed that, thanks. Should be fixed now, fixedup/squashed into the original commit.